### PR TITLE
New validation process

### LINKED ISSUE #263: Re-design of the Validation process
Implement a re design of the verification process. 

-add a direct notification of who trusted you
**Design Material**
_Marked in green_ https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/Circles-myxogastria?node-id=4350%3A18052

### LINKED ISSUE #264: Re-designing landing page of the wallet
New design of Landing page. It has 3 levels of engagement:
1_Trust people and send circles
2_ ... Menu ( quick enries to edit profile, marketplace, support)
3_ Burger menu ( stays at it was) 

**Deisng Material**
_Marked in brown:_ https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/Circles-myxogastria?node-id=4356%3A49547

### LINKED ISSUE #323: Change Trust and Untrust box
https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/Circles-myxogastria?node-id=4932%3A27498

### PR DESCRIPTION
**Pull request by mikozet**
Hi @louilinn, 

List of changes:
Change Trust and Untrust box: https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/Circles-myxogastria?node-id=4932%3A27498
Re-designing landing page of the wallet: https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/Circles-myxogastria?node-id=4356%3A49547
Re-design of the Validation process: https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/Circles-myxogastria?node-id=4350%3A42078

You can find those tasks on the right side under 'Development' tag

I have created additional task Regarding Re-design of the Validation process as there is a missing feature 
https://github.com/CirclesUBI/circles-myxogastria/issues/325

### CODE REVIEW

File: src/components/OnboardingStepper.js [hunk 1]
```diff
@@ -176,15 +176,19 @@ const OnboardingStepper = ({
...
-        <Button
-          disabled={isDisabled}
-          fullWidth
-          isPrimary
-          onClick={isLastSlide ? onFinish : onNext}
-        >
-          {stepsButtons[current].btnNextStep}
-        </Button>
+        {stepsButtons[current].additionalBtnSecond && (
+          <CopyToClipboardBtn onClick={onFinish} />
```
Comment (louilinn) [inline src/components/OnboardingStepper.js:180]: What is the intention here?

File: src/views/Validation.js [hunk 2]
```diff
@@ -30,69 +31,145 @@ const useStyles = makeStyles((theme) => ({
...
-            <Button fullWidth isPrimary to={VALIDATION_SHARE_PATH}>
+            <Button fullWidth isOutline onClick={shareProfileBtnHandler}>
               {translate('Validation.buttonShareProfileLink')}
             </Button>
           </Box>
         )}
+        <Fragment>
+          <Box mb={0} mt={2}>
+            <Button
+              disabled={isReady ? false : true}
```
Comment (louilinn) [inline src/views/Validation.js]: you could simply write `!isReady`

File: src/views/Validation.js [hunk 3]
```diff
@@ -30,69 +31,145 @@ const useStyles = makeStyles((theme) => ({
...
   useUpdateLoop(async () => {
     await dispatch(checkTrustState());
   });
 
-  const leftTrustConnections = Math.max(
-    0,
-    NEEDED_TRUST_CONNECTIONS - trust.connections,
-  );
-
   const isDeploymentReady =
```
Comment (louilinn) [inline src/views/Validation.js:85]: what is the difference between isDeploymentReady and isReady? Can we make names clearer, in the comment below "Attempt deployment if one of two conditions is met" makes them sound the same based on naming

File: locales/en.json [hunk 4]
```diff
@@ -369,7 +374,7 @@
     "infoUbiPayoutReceived": "You received {payout} Circles!"
   },
   "Validation": {
-    "bodyDoNotReset": "Do not reset your browser or you will need to recover your data with your seed phrase.",
+    "bodyDoNotReset": "Do not reset your browser local storage or you will need to recover your data with your seed phrase.",
     "bodyTrustDescription": "To receive your basic income,",
     "bodyTrustDescriptionEmphasize": "three people must trust your profile.",
     "buttonShareProfileLink": "Share Profile Link",
```
Comment (louilinn) [inline locales/en.json:380]: Not related to the implementation work but I think we should have a different text in place of "Create account" button which appears after you get a notification your account is created @triaslucia

File: src/components/ValidationStatus.js [hunk 5]
```diff
@@ -44,25 +39,20 @@ const useStyles = makeStyles((theme) => ({
 
 const ValidationStatus = () => {
   const classes = useStyles();
-  const dispatch = useDispatch();
   const { app, safe, trust } = useSelector((state) => state);
 
+  // Safe and Token is already deployed or being deployed right now?
+  if (app.isValidated || safe.pendingIsLocked) {
+    return null;
```
Comment (louilinn) [inline src/components/ValidationStatus.js]: is this what we want?

**Review by louilinn — COMMENTED** [review]
These are my comments so far. I will also post some screenshots!

File: locales/en.json (hunk 4 above)
Comment (louilinn) [reply locales/en.json:380]: eg "Start using Circles" or "Finalize account" etc

**Comment by louilinn** [comment]
This is different than the design

Screenshot from 2022-05-03 16-00-39
Screenshot from 2022-05-03 16-01-36
Screenshot from 2022-05-03 16-02-40
Screenshot from 2022-05-03 16-13-26

**Comment by louilinn** [comment]
The padding of dialogues seems inconsistent

Screenshot from 2022-05-03 16-11-58
Screenshot from 2022-05-03 16-04-59

**Comment by louilinn** [comment]
There is an issue with buttons when closing dialogues or going back, highlighting gets stuck somehow

Screenshot from 2022-05-03 15-38-36
Screenshot from 2022-05-03 15-48-38

**Review by louilinn — CHANGES_REQUESTED** [review]
Requested changes based on screenshots

File: src/components/OnboardingStepper.js (hunk 1 above)
Comment (mikozet) [reply src/components/OnboardingStepper.js:180]: So stepsButtons is a configuration object where we decide which buttons should be displayed. So once there is an attribute `additionalBtnSecond` we tell which component should be displayed in this particular place. Here we check for that attribute and display component

File: src/views/Validation.js (hunk 2 above)
Comment (mikozet) [reply src/views/Validation.js]: corrected

File: src/views/Validation.js (hunk 3 above)
Comment (mikozet) [reply src/views/Validation.js:85]: I removed `isReady` - it was remainings from old code which I believe we don't need

File: locales/en.json (hunk 4 above)
Comment (mikozet) [reply locales/en.json:380]: Good point, let's talk about it with Lucia on UX meeting

File: src/components/ValidationStatus.js (hunk 5 above)
Comment (mikozet) [reply src/components/ValidationStatus.js]: I removed it as I believe it is redundant - it is from old codebase..

**Comment by mikozet** [comment]
> The padding of dialogues seems inconsistent
> 
> Screenshot from 2022-05-03 16-11-58 Screenshot from 2022-05-03 16-04-59

I corrected it

**Comment by mikozet** [comment]
> There is an issue with buttons when closing dialogues or going back, highlighting gets stuck somehow
> 
> Screenshot from 2022-05-03 15-38-36 Screenshot from 2022-05-03 15-48-38

I am not able to replicate it - could you describe what do you do?

**Comment by louilinn** [comment]
> 

I open a dialog then close it and then it happens, or going back a step. Let's see if it remains next week when we meet Lucia

File: src/components/ValidationStatus.js [hunk 6]
```diff
@@ -39,12 +39,7 @@ const useStyles = makeStyles((theme) => ({
 
 const ValidationStatus = () => {
   const classes = useStyles();
-  const { app, safe, trust } = useSelector((state) => state);
-
-  // Safe and Token is already deployed or being deployed right now?
-  if (app.isValidated || safe.pendingIsLocked) {
-    return null;
-  }
```
Comment (louilinn) [inline src/components/ValidationStatus.js]: why was it redundant?

**Comment by louilinn** [comment]
> This is different than the design
> 
> Screenshot from 2022-05-03 16-00-39 Screenshot from 2022-05-03 16-01-36 Screenshot from 2022-05-03 16-02-40 Screenshot from 2022-05-03 16-13-26

So what about this?

**Comment by mikozet** [comment]
> 

I have created additional task Regarding Re-design of the Validation process as there is a missing feature
https://github.com/CirclesUBI/circles-myxogastria/issues/325

**Review by louilinn — APPROVED** [review]
Complementing changes in this PR https://github.com/CirclesUBI/circles-myxogastria/issues/325
All good, lets review the entirety with Lucia later in preview. 
Great work! :star:

**Pull request merged**